app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -50,9 +50,6 @@ class RouteModeView : LinearLayout, RouteViewController, ViewPager.OnPageChangeL
     val mapListToggle: MapListToggleButton by lazy {
         findViewById(R.id.map_list_toggle)as MapListToggleButton
     }
-    val routeCancelButton: ImageView by lazy {
```

Comment:
@sarahlensing and I discussed this some. Currently, 'x' in the nav map view exits nav and list view just collapses list. We both agreed that exiting from list view is not ideal for users and having the 'x' do different things on different screens is not good either.
So… the 'x' is no more available on the list view screen. We noticed that Google doesn't have it either. What are your thoughts @ecgreb?